student.rb

```diff
@@ -6,6 +6,7 @@ class Student < Person
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.students << self
```

Comment:
Hi @Shakir-Hussain12 
- [ ] Please add a classroom method to the Student class that checks for a student belongs-to a classroom
[Guide](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)